yorkie/src/main/kotlin/dev/yorkie/util/YorkieException.kt

```diff
@@ -7,36 +7,36 @@ import kotlin.contracts.contract
  * `YorkieError` is an error returned by a Yorkie operation.
  */
 public data class YorkieException(public val code: Code, public val errorMessage: String) : RuntimeException(errorMessage) {
-    public enum class Code {
+    public enum class Code(val codeString: String) {
```

Comment:
I thought similarly, but I implemented it by referring to the JS SDK. 
I agree with your suggestion because it's more simpler, so I'll modify it if it doesn't make a big difference in the future (maybe I should start the first letter of the enum with a lowercase letter).